README.md

```diff
@@ -113,22 +150,28 @@ run `cabal` in a terminal, your shell will run the up-to-date version in your
 home directory. You will want to add the following lines to your shell's
 configuration file:
 
-    PATH=$PATH:$HOME/.cabal/bin
-    alias cabal="$HOME/.cabal/bin/cabal"
+```bash
+PATH=$PATH:$HOME/.cabal/bin
+alias cabal="$HOME/.cabal/bin/cabal"
+```
 
 If you don't know what your shell is, more than likely, your shell is Bash. If
 you use Bash, the file you will edit is `~/.bashrc`. If you use Z-shell, the
 file is `~/.zshrc`. You can run the following command to find out what your
 shell is.
 
-    < /etc/passwd grep $(whoami) | awk -F/ '{print $NF}'
+```bash
+echo $SHELL | xargs basename
```

Comment:
I'm don't consider myself a shell expert, so you might want to see whether both the commands work the same.